e2e/app.e2e-spec.ts

```diff
@@ -0,0 +1,15 @@
+import { Page } from './app.po';
+
+describe('App', () => {
+  let page: Page;
+
+  beforeEach(() => {
+    page = new Page();
+  });
+
+  describe('default screen', () => {
+    beforeEach(() => {
+      page.navigateTo('/');
+    });
+  });
+});
```

Comment:
You should also test for page title here :)

Reply:
oh ok, we thought that was just an example test, we will make sure to do that

Reply:
Has this been done? any plans on doing that on this PR? or a separate one?